src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -191,7 +191,7 @@ def view_user(conn, user):
         rows = cur.fetchall()
         headers = [description[0] for description in cur.description]  # column names
         if not rows:
-            return "User not found in association_table"
+            raise ValueError("User not found in association_table")
```

Comment:
Suggestion: Not a requirement, but it might be nice to include the user name (or is `user` a userid) in the error string:

```python
raise ValueError(f"User {user} not found in association table")
```

Reply:
That's a great suggestion! I like that. I just force-pushed up a fix to include the username in the error string, along with all of the other respective names (bank, queue, project) in their error strings.